packages/flutter/test/material/app_test.dart

```diff
@@ -117,4 +117,16 @@ void main() {
     expect(buildCounter, 2);
   });
 
+  testWidgets('Cannot pop the initial route', (WidgetTester tester) async {
```

Comment:
Nope.  The first one (in the Route base class) is really more documentation that code.  We don't have any non-abstract Route subclasses that don't override that method.